src/main/scala/uk/gov/nationalarchives/db/users/Lambda.scala

```diff
@@ -0,0 +1,31 @@
+
+package uk.gov.nationalarchives.db.users
+
+import scalikejdbc._
+import uk.gov.nationalarchives.db.users.Config._
+
+import java.io.{InputStream, OutputStream}
+import java.nio.charset.Charset
+
+class Lambda {
+
+  def process(inputStream: InputStream, outputStream: OutputStream) = {
+    println("Running")
```

Comment:
Is the `println` statement needed?

Reply:
No, I was debugging I think, I've taken it out.